tests/testsuite/fixture.go

```diff
@@ -120,6 +120,26 @@ func addTestAnnotation(vmi *v1.VirtualMachineInstance) {
 	vmi.Annotations["kubevirt.io/created-by-test"] = GinkgoT().Name()
 }
 
+func nfsAntiAffinity(vmi *v1.VirtualMachineInstance) {
+	for _, s := range CurrentSpecReport().Labels() {
+		if s == "RequiresRWXFsVMStateStorageClass" {
```

Comment:
Is this just for testing purposes? surely there are other tests without this label?

Reply:
All tests that require RWX FS should have the label. It's possible it's not the case yet, but we'll find them and fix them!

Reply:
This is the VM state label, no?

Reply:
No, the test label, i.e. decorator

Reply:
Indeed! That's pretty bad, I'll open a PR to add it to all the test that do RWX FS.

Reply:
I'm still confused. Why would every RWX FS test have a VM state decorator